src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -255,22 +263,18 @@ void remove_actor(SchedulingAlgorithmState *algorithm_state, ActorID actor_id) {
 
   /* Log some useful information about the actor that we're removing. */
   char id_string[ID_STRING_SIZE];
-  task_queue_entry *elt;
-  int count;
-  DL_COUNT(entry->task_queue, elt, count);
+  int64_t count = entry->task_queue->size();
```

Comment:
I'd just use `size_t` (or maybe `auto`, though I'd be careful where I sprinkle `auto`) here. No need to change the signedness unnecessarily.